TramsDataApi/UseCases/GetAcademyConversionProjects.cs

```diff
@@ -2,65 +2,36 @@
 using TramsDataApi.ResponseModels.AcademyConversionProject;
 
 namespace TramsDataApi.UseCases
 {
-    public class GetAcademyConversionProjects : IUseCase<GetAllAcademyConversionProjectsRequest, IEnumerable<AcademyConversionProjectResponse>>
+    public class GetAcademyConversionProjects : IGetAcademyConversionProjects
     {
         private readonly IAcademyConversionProjectGateway _academyConversionProjectGateway;
-        private readonly ITrustGateway _trustGateway;
         private readonly IEstablishmentGateway _establishmentGateway;
-        private readonly IIfdPipelineGateway _ifdPipelineGateway;
 
         public GetAcademyConversionProjects(
             IAcademyConversionProjectGateway academyConversionProjectGateway,
-            ITrustGateway trustGateway, 
-            IEstablishmentGateway establishmentGateway,
-            IIfdPipelineGateway ifdPipelineGateway)
+            IEstablishmentGateway establishmentGateway)
         {
             _academyConversionProjectGateway = academyConversionProjectGateway;
-            _trustGateway = trustGateway;
             _establishmentGateway = establishmentGateway;
-            _ifdPipelineGateway = ifdPipelineGateway;
         }
 
-        /// <remarks>
-        /// May make more sense to add trust and sponsor info into ACP table when an ACP is created so that no need to query trusts table
-        /// Just not sure if a trust or sponsor name could change while an ACP is still open
-        /// </remarks>
-        public IEnumerable<AcademyConversionProjectResponse> Execute(GetAllAcademyConversionProjectsRequest request)
+         public IEnumerable<AcademyConversionProjectResponse> Execute(int page, int count)
         {
-            var ifdProjects = _ifdPipelineGateway.GetPipelineProjects(request.Page, request.Count)
-                .ToList();
-
-            var academyConversionProjects = _academyConversionProjectGateway
-                .GetByIfdPipelineIds(ifdProjects.Select(i => i.Sk).ToList()).ToList();
-            
-            var trustRefs = academyConversionProjects
-                .Where(acp => !string.IsNullOrEmpty(acp.TrustReferenceNumber))
-                .Select(acp => acp.TrustReferenceNumber)
-                .ToArray();
-
-            var trusts = _trustGateway
-                .GetIfdTrustsByTrustRef(trustRefs)
-                .Select(t => new { t.TrustRef, TrustName = t.TrustsTrustName })
-                .ToArray();
-
-            var responses = academyConversionProjects
-                .Where(p => !string.IsNullOrEmpty(p.SchoolName))
-                .Select(p => AcademyConversionProjectResponseFactory
-                    .Create(p, null, ifdProjects.FirstOrDefault(ifd => ifd.Sk == p.IfdPipelineId)))
-                .ToList();
-
+            var conversionProjects = _academyConversionProjectGateway.GetProjects(page, count);
+                      
+            var responses = conversionProjects
+                .Where(acp => acp.SchoolName != null)
```

Comment:
Do we need to think about what these schoolName == null filters are about and whether we want them?

Reply:
Will remove this, it appears that this was a workaround to ignore incomplete projects in an earlier iteration.  This should not happen with the new project creation approach